src/components/Header.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+
+export default class Header extends Component {
+    render() {
+        return (
+            <>
+            <header id="nav-bar">
+            <div id="left-side-nav"> 
+                <img class="logo" src="logo.png" alt = "logo"/>
+                <div>
+                    <h1>{this.props.heading}</h1>
+                    <h6>the Best Food Blog on The Web</h6>
+                </div>
+            </div>
+            <div id="right-side-nav">
```

Comment:
Everything from line 15-24 should be refactored into ButtonList, and then call ButtonList here.